.github/workflows/docker-build-push-github.yaml

```diff
@@ -7,6 +7,15 @@ name: Package OCI
 on:
   # This entry allows calling this workflow from other workflows
   workflow_call:
+    inputs:
+      docker-build-context:
+        description: 'Build context is the PATH or URL to build from. '
+        type: string
+        default: .
+      docker-file-path:
```

Comment:
I know you're like to be verbose, and it's usually a right call, but I fail to see the value on doing that. Why don't we call this `path` instead?

Reply:
I'm probably future-proofing for no good reason, but I find the `docker` prefix inoffensive and `file` is actually the parameter used by the `docker/build-push-action` action, which I didn't find intuitive enough, which is why I added the `path` suffix.

I can drop the `path` suffix if you want.

Reply:
Let's use `dockerfile` or `filepath`, whatever you think is best.